packages/components-js/tests/vrt/puppeteer/specs/pagination.vrt.ts

```diff
@@ -1,5 +1,44 @@
-import { defaultViewports, getVisualRegressionTester, vrtTest } from '@porsche-design-system/shared/testing';
+import {
+  defaultViewports,
+  getVisualRegressionStatesTester,
+  getVisualRegressionTester,
+  vrtTest,
+} from '@porsche-design-system/shared/testing';
+import {
+  forceFocusHoverState,
+  forceFocusState,
+  forceHoverState,
+  getThemedBodyMarkup,
+  GetThemedMarkup,
+  setContentWithDesignSystem,
+} from '../helpers';
+import type { Theme } from '@porsche-design-system/utilities-v2';
 
-xit.each(defaultViewports)('should have no visual regression for viewport %s', async (viewport) => {
+it.each(defaultViewports)('should have no visual regression for viewport %s', async (viewport) => {
   expect(await vrtTest(getVisualRegressionTester(viewport), 'pagination', '/#pagination')).toBeFalsy();
 });
+
+it('should have no visual regression for :hover + :focus-visible', async () => {
+  const vrt = getVisualRegressionStatesTester();
+  expect(
+    await vrt.test('pagination-states', async () => {
+      const page = vrt.getPage();
+
+      const getElementsMarkup: GetThemedMarkup = (theme: Theme) => `
+        <p-pagination total-items-count="500" items-per-page="25" active-page="1" theme="${theme}"></p-pagination>
+
+        <p-pagination total-items-count="500" items-per-page="25" active-page="2" theme="${theme}"></p-pagination>`;
+
+      await setContentWithDesignSystem(
+        page,
+        getThemedBodyMarkup(getElementsMarkup, { themes: ['light', 'dark', 'light-electric'] })
+      );
+
+      await forceHoverState(page, '.hover > p-pagination >>> span');
```

Comment:
Is the result of this in the screenshot really how it should look like with the different sizes etc?
Also why do "dead" spans without any interaction like the disabled one, the one with the `...` visually look alive?